src/libraries/Microsoft.Extensions.HostFactoryResolver/src/Microsoft.Extensions.HostFactoryResolver.Sources.csproj

```diff
@@ -4,5 +4,11 @@
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <!-- Avoid trying to binplace non existent documentation file.-->
     <EnableBinPlacing>false</EnableBinPlacing>
+    <IsShipping>false</IsShipping>
```

Comment:
I think the comment in L2 isn't valid anymore. (can't comment on it in the PR, which is why I chose this line)